packages/drawer/src/components/Drawer.tsx

```diff
@@ -127,6 +134,10 @@ class Drawer extends React.Component<DrawerProps, DrawerState> {
                   this.state.isOpen ? 'is-open' : 'is-closed',
                   this.props.className
                 )}
+                role="dialog"
+                aria-modal="true"
+                aria-labelledby={this.ids.labelledBy}
+                aria-describedby={this.ids.describedBy}
```

Comment:
We only need `aria-labelledby` for modal/dialog boxes. See: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute